test/parallel/xyk-pallet.API.pool.test.ts

```diff
@@ -434,3 +436,133 @@ describe("xyk-pallet - Pool tests: a pool can:", () => {
     );
   });
 });
+
+describe("xyk-pallet - Pool opeations: Simmetry", () => {
+  let testUser1: User;
+  let sudo: User;
+
+  let keyring: Keyring;
+  let firstCurrency: BN;
+  let secondCurrency: BN;
+
+  beforeAll(async () => {
+    try {
+      getApi();
+    } catch (e) {
+      await initApi();
+    }
+
+    await waitNewBlock();
+    keyring = new Keyring({ type: "sr25519" });
+
+    // setup users
+    testUser1 = new User(keyring);
+
+    sudo = new User(keyring, sudoUserName);
+
+    //add two curerncies and balance to testUser:
+    [firstCurrency, secondCurrency] = await Assets.setupUserWithCurrencies(
```

Comment:
I know this is a common pattern elsewhere, but would you be happy to use 'real' names in place of `firstCurrency` and `secondCurrency` such as BTC and ETH so it's easier to read?